paper-tooltip.html

```diff
@@ -183,27 +186,32 @@
       },
 
       hide: function() {
+        // prefer focus over mouseleave
+        if (this.target.focused) return;
+
         this._showing = false;
         this.playAnimation('exit');
       },
 
       _setPosition: function() {
-        if (!this._target)
-          return;
+        var target = this.target;
```

Comment:
The idea here is that I don't want to called `this.target` every time the tooltip is show, as this isn't good for performance